pkg/debug/toggle.go

```diff
@@ -0,0 +1,211 @@
+package debug
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/ulid"
+	"github.com/kolide/launcher/ee/socket"
+	"github.com/kolide/launcher/pkg/backoff"
+)
+
+type DebugToggleListener struct {
```

Comment:
I can roll with this, but reading it, it feels a little weird. 

Seeing it in code, it feels weird to imagine running `toggle`, instead I kinda imagine running `debug start`, and having that return a URL`. and `debug stop`.

With `toggle` the behavior feels much less clear. How does one check state? Or see what URL it's on? Etc.

Reply:
I kinda had that gut feeling too. When I got there it was already a toggle function and for some reason I thought I should keep the pattern. I will update it to have start / stop / status? sub commands

Reply:
My inclination is that both `start` and `status` should display the listening port.